app/models/rook.rb

```diff
@@ -1,2 +1,8 @@
 class Rook < Piece
+  def valid_move?(new_x, new_y)
```

Comment:
@Nikkovichert each line is a condition by itself.. so you'd need to connect them with && probably.